scanner/laravel.go

```diff
@@ -27,8 +25,6 @@ func configureLaravel(sourceDir string, config *ScannerConfig) (*SourceInfo, err
 		return nil, nil
 	}
 
-	files := templates("templates/laravel")
```

Comment:
@rubys The `templates/laravel` directory contain the `/.fly` directory. This directory contains default( _like cache.sh that configures the app's cache config file, etc_ ), and [custom startup scripts](https://fly.io/docs/laravel/the-basics/customizing-deployments/#startup-scripts) alike that the user can create, that are run by entrypoint.sh. 

I'm not sure if this should be generated by flyctl or by the dockerfile-laravel command

Reply:
I think that yes, it should be generated from the dockerfile-laravel command, I'ma try to list why in [this issue](https://github.com/orgs/fly-apps/projects/6?pane=issue&itemId=56706705).